markdown/Post-ProcessingGuides/SimpleTechniquesToFixColorBanding.md

```diff
@@ -0,0 +1,224 @@
+
+Now after creating a layer mask, click on it, and press Ctrl + I to invert that mask (layers should look like [this](https://i.imgur.com/ykrE66v.png)). What that will do is it will isolate the subject out of that effect. Lastly, grab a Brush by pressing B on a keyboard, or by picking up the Brush from the [left](https://i.imgur.com/sJWYYJ1.png) (make sure it’s a [black brush](https://i.imgur.com/YOhDeUQ.png)) with 100% opacity and 100% flow (by default it should be already), and paint around where you don’t want effects to be applied basically leaving the healthy areas of the image untouched by the surface blur and noise (if its the subject only that you want to be isolated then you don't have to use the brush, you can leave it). You can play with the opacity of the brush and flow to smooth out the transitions. [Here](https://i.imgur.com/RQF1iY6.png) you will see an example how everything is setup. In the banding area there is noise, and surface blur applied to fight the banding while other areas are untouched by the effects.
+
+
+Same goes for [Lightroom](https://framedsc.com/Post-ProcessingGuides/SimpleTechniquesToFixColorBanding.htm#Lightroom), just instead of sky select the background, and repeat the same steps as above with Subtract to mask, and use brushes to smooth out transitions and play with its settings.
+
+To break this down for Lightroom. 
+
+1. [example](https://i.imgur.com/YzWh5Cv.png) when Lightroom selected entire background. 
+2. [example](https://i.imgur.com/b9gOcZ0.png) when I used Subtract to Mask with a Brush to clean the areas so I am left with clean background selected only.
+3. [example](https://i.imgur.com/vVFDAxu.png) when I used a brush again to remove the background selection to focus only on color banding area. 
+4. Now you can apply the same edits as mentioned above to remove the banding.
+
+<br>  
+
+
+
+If the banding occurs duo to some lightning that bleeds out behind the subject, you can use Quick Selection Tool from Photoshop to select the banding area. Selection example in Photoshop should look like this [this](https://i.imgur.com/LI6uogy.png). Applied edits to remove the banding will only cover that area.
+
+
+In Lightroom, go to Masking (Shift+W) and select a Subject so the AI makes the selection for you. If there are two subjects and banding covers only one, then what you do is Subtract to Mask with a Brush to remove the selection from the other subject.
+Final selection example after doing the Subtract to Mask with a Brush in Lightroom looks like [this](https://i.imgur.com/Cu2SQYJ.png). Applied edits to remove the banding will only cover that area.
+
+
+<img src="../Images/PostProcessingGuides/qbbeforeandafterexample.png" /><div class="figure"><p>Smoothing out the banding that comes from the lighting which bleeds out behind.</p></div><br> 
+
+
+##Grain usage on shots in general, and why grain, noise and blur
```

Comment:
Add a space between the ## and the title so that it gets rendered as an h2.

Reply:
Hey sup! I am glad to hear from you really. I know you are from Otis server , I mean I recognize the name. :)
Anyway, yeah lets make this guide shine even more! The only thing is that trust me, even though you can probably write it in 1 day or so, I am first time doing something like this, and I don't have any knowledge in HTML, I didn't even know whats the markdown etc. I have zero skills in programming, I tried to do everything by myself without bothering anyone, and I'm tired haha, so even if everything looks great to me I can understand that it might not look okay for someone else or for the site moderators while reviewing it. So thanks for the feedback, I am open and willing to improve on this, because I am planning to write more guides in the future.

Ngl, your answer feels like writting a guide once again lmao, ahah :P

I'm gonna be short as much as I can to answer:

1. Oops, I saw from one guy how he didn't had that in actual folder, but idk., I left it there because I thought its important for the first time, and I thought it won't impact you.
2. I actually compressed the images even more via FastStone, but I got some weird lines on them, to be precise on the gradient, so idk, I thought that 38mb in total wouldn't be too much so I left them like that. Will see what I can do. 
3. Ok, cool
4. I used DebandShader couple of times, but I'm still not quite good with it. Usually my post is more focused outside of the game. I do use ReShade a lot, but thats for stuff like HeightFog, LUTs, DOF and similar, and even after that I still edit it in Lightroom heavily. Will see what I can potentially write about that.
5. Will do!
6. Will do as well!
7. Oki
8. Oki 2
9. AI models, I'm not familiar with that at all, so be my guest! 
10. The reason why so many imgur links is exactly because of that. But tbh I haven't even try to see how it would look on the site without links.
11. Thats amazing! You see I needed to go in some game to actually shot, but I was so tired so I was like I can update it in the future and add stuff. But thats great if you have an example already, so again, be my guest! 
12. You see I never heared of that plugin, so I am curious to try that in the future. I do have Nik Collection 6 plugins, but I never heared of the one you use. Okay no problem. 


Gonna chill for the weekend, but I will also slowly do some changes, some fast changes. Then I start to cover other sections and see what I can do there.

Reply:
Don't worry about it! Using Github in itself is already a hard task for the average person, don't even mind the whole programming aspect behind HTML and markdown if you are not familiar with it. So appreciate the time and hard work you put into this even more! Besides the things I pointed out you did great! 

And please! I absolutely don't mind helping people out to contribute to the site. At the end of the day, it's useful to us all. So if you need help with any of the things I pointed out do not hesitate to send me a DM on Discord.

Regarding point 4, don't you worry about writing much. Just mentioning it in a single sentence is enough.

And please, don't worry about taking the time to do this! At the end of the day it is something we do because we want to, not because we are paid to do so, so relax and take your time with it :)

Reply:
Its indeed tricky. Luckily I kinda adapt fast to something. And there are GitHub help tutorials etc. Also stuff that Frans wrote it. Still lots of reading to do, and to test back and fourth, but I don't mind that. Thanks a lot!

True. Though I hate myself for being perfectionist, so even if its for free, I like to do it fast, and with passion. Kinda I like full release to be fast and without bugs, which is impossible I guess right haha.

Sure I'm gonna hit you up on discord if I need you for something.